src/Componets/MainPage/shared/error.js

```diff
@@ -4,6 +4,5 @@ import React from 'react';
 export const Error = props =>(

```

Comment:
add styles to this component, e.g. Red text, or will good do it like push notification ,e.g. 
you can use this library for https://www.npmjs.com/package/react-notifications-component

Reply:
I use simple css or better use library react-notifications-component?

Reply:
better use library react-notifications-component